examples/gcp/cloud-sql/rg.yaml

```diff
@@ -0,0 +1,197 @@
+apiVersion: kro.run/v1alpha1
+kind: ResourceGroup
+metadata:
+  name: cloudsql.kro.run
+spec:
+  schema:
+    apiVersion: v1alpha1
+    kind: CloudSQL
+    spec:
+      name: string
+      project: string
+      primaryRegion: string
+      replicaRegion: string
+    status:
+      connectionName: ${sqlPrimary.status.connectionName}
+      ipAddress: ${sqlPrimary.status.firstIpAddress}
+  resources:
+  - id: cloudkmsEnable
+    template:
+      apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+      kind: Service
+      metadata:
+        annotations:
+          cnrm.cloud.google.com/deletion-policy: "abandon"
+          cnrm.cloud.google.com/disable-dependent-services: "false"
+        name: cloudkms-enablement
+      spec:
+        resourceID: cloudkms.googleapis.com
+  - id: iamEnable
+    template:
+      apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+      kind: Service
+      metadata:
+        annotations:
+          cnrm.cloud.google.com/deletion-policy: "abandon"
+          cnrm.cloud.google.com/disable-dependent-services: "false"
+        name: iam-enablement
+      spec:
+        resourceID: iam.googleapis.com
+  - id: servceusageEnable
```

Comment:
Nit: serviceUsageEnable

Reply:
It would also be nice to make this optional, one less thing to worry about when writing a template.  Obviously we couldn't refer to "unnamed variables" ... but maybe it is important for a stable status?